modular_skyrat/modules/ashwalkers/code/buildings/railroad.dm

```diff
@@ -0,0 +1,127 @@
+/obj/item/stack/rail_track
```

Comment:
You need to add a `singular_name` variable that represents the name of a singular railroad track, and a `merge_type` variable that I'm not quite sure the usage of, but I'll let you figure that one out.